tests/test_errata.py

```diff
@@ -422,6 +422,82 @@ def test_errata_get_cve_affected_rpm_nvrs(self, errata_http_get, errata_rest_get
         ret = self.errata.get_cve_affected_rpm_nvrs(28484)
         self.assertEqual(ret, ['libntirpc-1.4.3-4.el6rhs'])
 
+    @patch.object(Errata, "_errata_rest_get")
+    @patch.object(Errata, "_errata_http_get")
+    def test_errata_get_cve_affected_build_nvrs(self, errata_http_get, errata_rest_get):
+        mocked_errata = MockedErrataAPI(errata_rest_get, errata_http_get)
+        mocked_errata.builds_by_cve = {
+            "CVE-2021-42574": {
```

Comment:
I'd suggest using some fake data instead of these real data which may have some security risks.

Reply:
Good suggestion, fixing.